script/scaffold/templates/config_flow/integration/config_flow.py

```diff
@@ -64,22 +64,31 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     async def async_step_user(self, user_input=None):
         """Handle the initial step."""
+        if user_input is None:
+            return self.async_show_form(
+                step_id="user", data_schema=STEP_USER_DATA_SCHEMA
+            )
+
         errors = {}
-        if user_input is not None:
-            try:
-                info = await validate_input(self.hass, user_input)
-
-                return self.async_create_entry(title=info["title"], data=user_input)
-            except CannotConnect:
-                errors["base"] = "cannot_connect"
-            except InvalidAuth:
-                errors["base"] = "invalid_auth"
-            except Exception:  # pylint: disable=broad-except
-                _LOGGER.exception("Unexpected exception")
-                errors["base"] = "unknown"
+
+        try:
+            info = await validate_input(self.hass, user_input)
+
```

Comment:
I think it's a bit easier to read if there are no blank lines in the try... except...else block. Just have a blank line before and after that block to show that that block belongs together.